client/src/components/exercise/lab10/pages/BuildingAIRepair.js

```diff
@@ -0,0 +1,130 @@
+import React, { Component, Fragment } from "react";
+import { EXERCISE_PLAYING } from "../../../../constants";
+import { actions as repairActions } from "../../../../reducers/lab10/RepairReducer";
+import { actions as exerciseActions } from "../../../../reducers/lab10/ExerciseReducer";
+import { bindActionCreators } from "redux";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+import Popup from "src/components/all-components/Popup";
+import BuildingAICodeBlock from "../components/code/BuildingAICodeBlock";
+import { navigate } from "@reach/router";
+
+class BuildingAIRepair extends Component {
```

Comment:
why class components?

Reply:
Sorry, that was probably me or Janae from spring semester. Didn't figure out the functional thing til summer time lol

Reply:
yeah, i was just building upon what already existed

Reply:
no worries just wanted to clarif, probably a tech debt thing later